frontend/app/templates/work_packages/tabs/_revision_activity.html

```diff
@@ -0,0 +1,22 @@
+<div class="work-package-details-activities-activity-contents">
+  <div class="comments-number">
+    <span ui-sref="work-packages.list.details.activity.details({ activity_no: activityNo })">
+      <a name="{{ activityNo }}"
+         ng-bind="'#' + activityNo"
+         href="#{{ activityNo }}"></a>
+    </span>
+  </div>
+  <img class="avatar" ng-src="{{ userAvatar }}" alt="Avatar" title="{{userName}}" ng-if="userAvatar" />
```

Comment:
Using `ng-if` with an image can flicker shortly in IE10 and IE11, as the broser attempts to load the image initialyy, better to wrap it into a `div` imho.